NextGenConnect/JSON to HL7v2.xml

```diff
@@ -253,21 +244,40 @@ var location = msg[&apos;entry&apos;].filter(function (entry){
     return entry[&apos;resource&apos;][&apos;resourceType&apos;] === &quot;Location&quot;;
 })[0][&apos;resource&apos;];
 
+/* Need to add these to the bundle
+var organization = msg[&apos;entry&apos;].filter(function (entry){
+    return entry[&apos;resource&apos;][&apos;resourceType&apos;] === &quot;Organization&quot;;
+})[0][&apos;resource&apos;];
+
+var actor = msg[&apos;entry&apos;].filter(function (entry){
+    return entry[&apos;resource&apos;][&apos;resourceType&apos;] === &quot;Practioner&quot;;
+})[0][&apos;resource&apos;];
+*/
 
 //RXA-1 Must be 0
 tmp[&apos;RXA&apos;][&apos;RXA.1&apos;] = 0;
 
 //RXA-2 Must be 1
 tmp[&apos;RXA&apos;][&apos;RXA.2&apos;] = 1;
 
-//RXA-3 Data of Administration -- Need to worry if time component stored 
-//tmp[&apos;RXA&apos;][&apos;RXA.3&apos;] = immunization[&apos;occurenceDateTime&apos;].replace(&apos;-&apos;,&apos;&apos;); // commented out due to undefined
+//RXA-3 Data of Administrationmsg[&apos;entry&apos;][0][&apos;resource&apos;][&apos;occurrenceDateTime&apos;]
+
+var occurenceDate = immunization[&apos;occurrenceDateTime&apos;];
+tmp[&apos;RXA&apos;][&apos;RXA.3&apos;] = occurenceDate.split(&apos;-&apos;).join(&apos;&apos;); 
 
-//RXA-5 Administered Code -- Needs to be CVX -- how are we converting what&apos;s stored in FHIR resource? Hardcode?
+//RXA-5 Administered Code -- // NDC code?
 
-// If/switch  statements hard code this in... cvx codes for coronavirus ....
-tmp[&apos;RXA&apos;][&apos;RXA.5.1&apos;][&apos;RXA.5.1.1&apos;]  = immunization[&apos;vaccineCode&apos;][&apos;coding&apos;][0][&apos;code&apos;];
-tmp[&apos;RXA&apos;][&apos;RXA.5.1&apos;][&apos;RXA.5.1.2&apos;]  = immunization[&apos;vaccineCode&apos;][&apos;coding&apos;][0][&apos;display&apos;];
+/* Need to add organization to bundle
```

Comment:
Any reason not to add these as part of this PR? Or are you leaving it for later? If so, is there a card in the backlog?

Reply:
There is a card to add the extra resources to the bundle in the backlog.

Reply:
Which card is it? Could you include a link to it in your reply?

Reply:
https://mass-immunization-system.atlassian.net/jira/software/projects/MVS/boards/2?selectedIssue=MVS-530

Reply:
This is fixed.